From c5911dc74ad3d2858962e3384753c49cc3ad5ebe Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Thu, 4 Apr 2019 12:32:26 +0200
Subject: [PATCH] nft.8: Update meta pkt_type value description

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1628694
Upstream Status: nftables commit d75e9184bb51a
Conflicts: Changes applied manually to doc/nft.xml, upstream switched to
           using asciidoc.

commit d75e9184bb51a1311ac950b13384f329836d597e
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Sep 14 11:00:14 2018 +0200

    nft.8: Update meta pkt_type value description

    Commit 8a7f6de536408 ("meta: fix pkttype name and add 'other' symbol")
    deprecated pkt_type value 'unicast' (for it being misleading) and
    introduced 'host' and 'other' but it did not update documentation
    accordingly. Fix this by replacing 'unicast' with 'host' in
    documentation and adding 'other'.

    While being at it, make sure these literal values are recognized as
    such: Put them in all lower-case (as required by the parser) and in bold
    font (to stand out a bit more).

    Signed-off-by: Phil Sutter <phil@nwl.cc>
    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
---
 doc/nft.xml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/doc/nft.xml b/doc/nft.xml
index a4a4c3f..12b6cea 100644
--- a/doc/nft.xml
+++ b/doc/nft.xml
@@ -2734,8 +2734,9 @@ filter output icmpv6 type { echo-request, echo-reply }
 							<row>
 								<entry>pkt_type</entry>
 								<entry>
-									Packet type: Unicast (addressed to local host),
-									Broadcast (to all), Multicast (to group).
+									Packet type: <literal>host</literal> (addressed to local host),
+									<literal>broadcast</literal> (to all), <literal>multicast</literal> (to group),
+									<literal>other</literal> (addressed to another host).
 								</entry>
 							</row>
 						</tbody>
-- 
1.8.3.1

